release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3009,7 +3009,42 @@ link:https://access.redhat.com/solutions/6114681[{product-title} 4.6.34 containe
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6128971[{product-title} 4.6.35 container image list]
+
+[id="ocp-4-6-35-features"]
+==== Features
+
+[id="ocp-4-6-35-cluster-operator-gatherer-enhancement"]
+===== Insights Operator enhancement
+
+With this update, ClusterOperator `Gather` can now collect logs from pods that are related to unhealthy operators, and/or pods that are living in the same namespace as the operator. This allows the Insights Operator to determine the namespace associated with the operator and include pod logs in the bundle. As a result, the Insights Operator is now collecting logs from unhealthy pods not only related to the operator, but from every pod that lives in the operator namespace.  
+
+[id="ocp-4-6-35-bug-fixes"]
+==== Bug fixes
+
+* Previously, when bringing up a single-stack IPv6 cluster on nodes with IPv4 address, kubelet might have used the IPv4 IP instead of the IPv6 IP for the node IP. Consequently, host network pods would have IPv4 IPs rather than IPv6 IPs, which made them unreachable from IPv6-only pods. This update fixes the node-IP-picking code, which results in the kubelet using the IPv6 IPs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942488[*BZ#1942488*])
+
+* The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=1953097[*BZ#1953097*] enabled the CoreDNS `bufsize` plugin with a size of 1232 bytes. Some primitive DNS resolvers are not capable of receiving DNS response messages over UDP that are greater than 512 bytes. Consequently, some DNS resolvers, such as Go's internal DNS library, are unable to receive long-winded DNS responses from openshift-dns. This update sets the CoreDNS `bufsize` to 512 bytes for all servers. As a result, UDP DNS messages are now properly received. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970140[*BZ#1970140*]) 
+
+* Previously, a lack of proper timeouts when executing http connections left connections opened. As a result, these connections accumulated until reaching the maximum limit, and consequently the operator would become incapable of processing incoming events. This update adds timeouts to the http client used by the operator, which ensures that open connections are closed once the timeout is reach. (link:https://bugzilla.redhat.com/show_bug.cgi?id=199563[*BZ#1959563*]) 
+
+* Previously, removing `selector` from a service exposed via a route resulted in the duplication of `endpointslices` that would be created for the service's pods, which would trigger HAProxy reload errors due to duplicate server entries. This update filters out accidental duplicate server lines when writing out the HAProxy config file, so that deleting the selector from a service no longer bricks the router. 
```

Comment:
Some alternative for "bricks" would be better. Maybe: "no longer causes the router to fail" or "no longer causes router errors"?

Reply:
BZ# and link is missing from this entry.